src/test/java/pl/project13/maven/git/GitDirLocatorTest.java

```diff
@@ -52,7 +52,9 @@ public void shouldUseTheManuallySpecifiedDirectory() throws Exception {
       assert foundDirectory != null;
       assertThat(foundDirectory.getAbsolutePath()).isEqualTo(dotGitDir.getAbsolutePath());
     } finally {
-      dotGitDir.delete();
+      if (!dotGitDir.delete()) {
+        dotGitDir.deleteOnExit();
+      }
```

Comment:
Mixed feelings here, probably not harmful here but had production trouble with deleteOnExit some years ago... Maybe we should log a warning instead?